src/main/java/com/taf/business/pages/selenium/BasePage.java

```diff
@@ -0,0 +1,38 @@
+package com.taf.business.pages.selenium;
+
+import com.taf.business.driver.DriverManager;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    WebDriver driver;
+
+    public BasePage(WebDriver driver) {
```

Comment:
Are you sure you should pass the driver as a param?

Reply:
You are right, it is not needed. It was deleted.